index.js

```diff
@@ -80,6 +84,12 @@ class Discovery extends EventEmitter {
       this.dht.on('peer', this._onDHTPeer)
       this._dhtAnnounce()
     }
+
+    if (opts.lsd === false) {
```

Comment:
Need to add `|| typeof LSD !== 'function'` for the browser case. The `browser: false` field you used in package.json tells Browserify to exclude the package and this is how to detect it.

Reply:
Fixed! 👍
I didn't know that, so learned for the next time 😉